ui/apps/platform/eslint-plugins/pluginGeneric.js

```diff
@@ -0,0 +1,139 @@
+/* globals module */
+
+const rules = {
+    // ESLint naming convention for positive rules:
+    // If your rule is enforcing the inclusion of something, use a short name without a special prefix.
+
+    'ExternalLink-a': {
+        // Require ExternalLink and a elements for consistent presentation.
+        // For example, ExternalLinkAltIcon follows a element.
+        meta: {
+            type: 'problem',
+            docs: {
+                description: 'Require ExternalLink and a elements for consistent presentation',
+            },
+            schema: [],
+        },
+        create(context) {
+            return {
+                JSXElement(node) {
+                    if (node.openingElement?.name?.name === 'a') {
+                        const ancestors = context.sourceCode.getAncestors(node);
+                        if (
+                            ancestors.length >= 1 &&
+                            ancestors[ancestors.length - 1].openingElement?.name?.name !==
+                                'ExternalLink'
+                        ) {
+                            context.report({
+                                node,
+                                message:
+                                    'Require ExternalLink and a elements for consistent presentation',
+                            });
+                        }
+                    }
+                },
+            };
+        },
+    },
+    'Link-target-rel': {
+        // Require props for consistent behavior and security of internal links that open in a new tab.
+        meta: {
+            type: 'problem',
+            docs: {
+                description:
+                    'Require if Link element has target="_blank" that it has rel="noopener noreferrer" props',
```

Comment:
nit: I think the wording could be clearer:

```Require that links with target="_blank" also have rel="noopener noreferrer"```

Reply:
Thank you again!